view/frontend/templates/promo/product-promotional-messaging.phtml

```diff
@@ -9,17 +9,22 @@ use Magento\Framework\Escaper;
  */
 
 $isEligibleProduct = $block->checkIsEligibleProduct();
+
+// The Amazon scripts will fail if there is more than a single class present for the selector
+$promoBannerClassConfig = explode(' ', $block->getPromoBannerClass());
+$promoBannerClass = $promoBannerClassConfig[0];
```

Comment:
@pmh1 What i found is that the amazon loaded script fails if more than a single selector is passed. must "class" and not "class1 class2" thus the explode. let me know if you think this is too heavy handed or would make more sense just being a static, hard coded value again! i think this works tho

Reply:
Also updated the config comment to highlight this limitation

Reply:
I'm wondering if this should be more of a dev-specific setting only so swapping out the admin-settings part but keeping everything else you have so that *if* someone wants to override it they can via this file.